### PR TITLE
fix(app-layout): add default spacing to AppPageLayout

### PR DESCRIPTION
**Pull request by portikM**
# Summary

Add default margin bottom to `AppPageLayout` component

## PR Checklist

* [ ] **Naming & Structure:** the files and package structure use the conventions outlined in the [Creating a Package docs](https://github.com/Kong/public-ui-components/blob/main/docs/creating-a-package.md).
* [ ] **Tests pass:** check the output of all package unit and/or component tests.
  * [ ] If this PR is the result of a bug, test coverage was added accordingly.
* [ ] **Functional:** all changes do not break existing APIs, but if so, a BREAKING CHANGE commit is in place to bump the major version.
* [ ] **Conventional Commits** all commits follow the conventional commit standards [outlined in the main README](https://github.com/Kong/public-ui-components#committing-changes).
* [ ] **Docs:** includes a technically accurate README, and the docs have been updated accordingly based on the changes in this PR.

### CODE REVIEW

File: packages/core/app-layout/src/components/pageHeader/AppPageHeader.vue [hunk 1]
```diff
@@ -86,6 +86,8 @@ const breadcrumbIconSlots = computed((): string[] => {
 
 <style lang="scss" scoped>
 .kong-ui-app-page-header {
+  margin-bottom: $kui-space-50;
```
Comment (adamdehaven) [inline packages/core/app-layout/src/components/pageHeader/AppPageHeader.vue:89]: I believe this should be 20px of margin bottom (`$kui-space-70`) based on the chat with Josh/Jason yesterday 

@jillztom does that sound correct?

Comment (portikM) [reply packages/core/app-layout/src/components/pageHeader/AppPageHeader.vue:89]: I used same spacing that was used in `khcp-ui` before that but I thin 20px would look much better.

Comment (portikM) [reply packages/core/app-layout/src/components/pageHeader/AppPageHeader.vue]: Updated to use `$kui-space-70`

Comment (adamdehaven) [reply packages/core/app-layout/src/components/pageHeader/AppPageHeader.vue]: If there is content in the bottom slot, does the bottom slot need default top margin? 

(Just commenting based on a convo yesterday; defer to Kai or Jilson for ✅ )

Comment (portikM) [reply packages/core/app-layout/src/components/pageHeader/AppPageHeader.vue]: No, below stol element has default margin top of `$kui-space-40`

**Review by jillztom — APPROVED** [review]

**Pull request merged**